Dev10/src/DoubleArrays.java

```diff
@@ -0,0 +1,68 @@
+import java.util.ArrayList;
+import java.util.Random;
+
+/**
+ * This class contains methods that works with arrays.
+ */
+public class DoubleArrays {
+
+  /**
+   * Initializes an array of arrays with random length and elements.
+   *
+   * @return an array of arrays
+   */
+  public double[][] initialize() {
+    Random random = new Random();
+    double[][] arrays = new double[random.nextInt(5)][];
+    for (int i = 0; i < arrays.length; i++) {
+      arrays[i] = new double[random.nextInt(5)];
+      for (int j = 0; j < arrays[i].length; j++) {
+        arrays[i][j] = random.nextDouble() * (random.nextInt(21) - 10);
+      }
+    }
+    return arrays;
+  }
+
+  /**
+   * Creates a new array which contains elements that belong at least two arrays.
+   *
+   * @param arrays array of arrays
+   * @return new array
+   */
+  public ArrayList createNewArray(double[][] arrays) {
+    ArrayList newArray = new ArrayList();
+    for (int i = 0; i < arrays.length - 1; i++) {
```

Comment:
Complexity of such approach is extremely high, this is not acceptable for production code